packages/router/package.json

```diff
@@ -19,6 +19,7 @@
     "build:watch": "nodemon --watch src --ext \"js,jsx,ts,tsx\" --ignore dist --exec \"yarn build\"",
     "prepublishOnly": "NODE_ENV=production yarn build",
     "test": "jest",
+    "test:types": "tstyche",
```

Comment:
Is it right to add scripts here? There might be some smarter way to run them via Lerna.

Reply:
Totally ok :) I would also add tstyche to the test script. i.e.

`"test": "jest && test:types"`

Reply:
Tried out your suggestion. It breaks `"test:watch": "yarn test --watch"` script. It I change `test` script, the `--watch` flag is passed to TSTyche not to Jest. Perhaps, let's leave it as is?

Reply:
Might be you want to move it. I added type tests to CI after Lint: https://github.com/redwoodjs/redwood/actions/runs/6847601897/job/18616200152?pr=9394

<img width="1041" alt="Screenshot 2023-11-13 at 10 51 34" src="https://github.com/redwoodjs/redwood/assets/72159681/e9c041a0-424f-4abb-847b-70510a26ae4b">